src/worker.cc

```diff
@@ -304,6 +305,9 @@ void add_date_to_locations(Options& options,
       case Options::arrive_by:
         locations.Mutable(locations.size() - 1)->set_date_time(options.date_time());
         break;
+      case Options::invariant:
+        for (auto& loc : locations)
+          loc.set_date_time(options.date_time());
```

Comment:
so, when its invariant, that means that all locations are computed at the same date_time